styles.css

```
@@ -9936,3 +10036,514 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
+    transform: translateX(clamp(0.75rem, 1.2vw, 1.4rem));
+}
+
+#fullscreen-cover-overlay .fullscreen-artwork-card {
+    width: min(420px, 100%);
+    aspect-ratio: 1 / 1;
+    border-radius: 18px;
+    overflow: hidden;
+    box-shadow: 0 28px 80px rgba(0, 0, 0, 0.26);
+}
+
+#fullscreen-cover-overlay #fullscreen-cover-image {
```

Stylelint (17.6.0)

[error] Expected selector "#fullscreen-cover-overlay `#fullscreen-cover-image`" to come before selector "#fullscreen-cover-overlay.controls-idle `#fullscreen-cover-image`", at line 4156 (no-descending-specificity)

(no-descending-specificity)